Assets/Scripts/ScriptableObjects/TestingControllers/PlayerController01.cs

```diff
@@ -0,0 +1,45 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+public class PlayerController01 : MonoBehaviour
```

Comment:
PlayerController01

not sure which to evaluate, it looks like playercontroller04 is the main one(?)

Reply:
Yes playercontroller04 is the main one, or one that we are currently using which is in the character controller folder, the others were tests till I found one that I liked which i put in the testing controllers folder, will probably delete or deal with these in the future, but will definitely be adding more testing scripts eventually, will be looking into ways of managing my testing scripts for the future.